components/lwip/Kconfig

```diff
@@ -111,6 +111,24 @@ config LWIP_DHCP_DOES_ARP_CHECK
         Enabling this option performs a check (via ARP request) if the offered IP address
         is not already in use by another host on the network.
 
+config LWIP_DHCPS_LEASE_UNIT
```

Comment:
Even though there's only two items here, what do you think about grouping these in a submenu?

Reply:
Added a submenu